test-commons/src/main/java/no/unit/nva/publication/PublicationGenerator.java

```diff
@@ -290,15 +297,19 @@ public static EntityDescription createSampleEntityDescriptionBook(URI bookSeries
             .withAbstract(randomString())
             .withAlternativeTitles(alternativeTitles)
             .withDescription(randomString())
-            .withLanguage(URI.create(LEXVO_ENG))
+            .withLanguage(randomLanguageUri())
             .withMetadataSource(randomUri())
             .withNpiSubjectHeading(randomString())
             .withTags(tags)
             .build();
     }
 
+    private static URI randomLanguageUri() {
+        return URI.create(LEXVO_ENG);
```

Comment:
Also not very random, again, I'd just use a few in an array

Reply:
Randomized